crypto/bio/bio.h

```diff
@@ -734,15 +734,6 @@ int BIO_hex_string(BIO *out, int indent, int width, unsigned char *data,
                    int datalen);
 
 struct hostent *BIO_gethostbyname(const char *name);
-/*-
- * We might want a thread-safe interface too:
- * struct hostent *BIO_gethostbyname_r(const char *name,
- *     struct hostent *result, void *buffer, size_t buflen);
- * or something similar (caller allocates a struct hostent,
- * pointed to by "result", and additional buffer space for the various
- * substructures; if the buffer does not suffice, NULL is returned
- * and an appropriate error code is set).
- */
```

Comment:
I don't see a reason to remove this comment.

Reply:
This comment was removed, when i introduced the BIO_gethostbyname_r function . In recent commit , i removed the function BIO_gethostbyname_r and forgot to bring back this comment in bio.h. 
Will follow up.